src/test/java/com/dabsquared/gitlabjenkins/publisher/GitLabAcceptMergeRequestPublisherTest.java

```diff
@@ -99,7 +99,7 @@ private HttpRequest prepareAcceptMergeRequest(String apiLevel, Long mergeRequest
             .withMethod("PUT")
             .withHeader("PRIVATE-TOKEN", "secret")
             .withHeader("Accept", "application/json")
-            .withHeader("User-Agent", "Jersey/2.40 (HttpUrlConnection 11.0.17)")
+            .withHeader("User-Agent", "Jersey/2.40 (HttpUrlConnection 11.0.20)")
```

Comment:
We cannot hard code this

Reply:
True, there is not such thing as default User-Agent string. But can we somehow obtain this from the system using code?

Reply:
Maybe we can try this?: https://www.mock-server.com/mock_server/creating_expectations.html

Reply:
Let's try to match by header regex value for `mock-server`: https://www.mock-server.com/mock_server/creating_expectations.html#button_match_request_by_header_value_regex